### PR TITLE
Launchdarkly Integration Tile

### PR DESCRIPTION
**Pull request by dsahni**
### What does this PR do?
Integration tile for the Launch Darkly integration

### CODE REVIEW

File: launchdarkly/manifest.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "launchdarkly",
+  "manifest_version": "1.0.0",
+  "maintainer": "support@launchdarkly",
+  "display_name": "LaunchDarkly",
+  "short_description": "Track metrics from LaunchDarkly's relay proxy",
+  "metric_prefix": "launchdarkly.",
+  "metric_to_check": "launchdarkly.test",
```
Comment (nmuesch) [inline launchdarkly/manifest.json]: Is the metric to check actually `launchdarkly.test`? I notice its not in the metadata.csv file, is this some healthcheck that the integration sends, and is this sent regularly/consistently?

File: launchdarkly/metadata.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+launchdarkly_relay.connections_env_platformCategory_userAgent,gauge,,connection,,The number of current proxied streaming connections.,0,launchdarkly,LaunchDarkly
+launchdarkly_relay.newconnections_env_platformCategory_relayId_userAgent,gauge,,connection,,The number of streaming connections created.,0,launchdarkly,LaunchDarkly
+launchdarkly_relay.requests_env_method_platformCategory_route_userAgent,gauge,,request,,Number of requests received.,0,launchdarkly,LaunchDarkly
```
Comment (nmuesch) [inline launchdarkly/metadata.csv]: I think the short names here should better reflect the metric themselves, for ex, the first one could be `relay conn` or something similar.

File: launchdarkly/manifest.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
```
[the remark this answers is not in the document]
Comment (dsahni) [reply launchdarkly/manifest.json]: Fixed 👍

File: launchdarkly/manifest.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  "maintainer": "support@launchdarkly",
+  "support": "support@launchdarkly",
+  "display_name": "LaunchDarkly",
+  "guid": "a1441ba8-be33-4123-8808-5a87cd696b64",
+  "short_description": "Track metrics from LaunchDarkly's relay proxy",
+  "metric_prefix": "launchdarkly_relay.",
+  "creates_events": true,
+  "public_title": "Datadog-LaunchDarkly Integration",
+  "type": "crawler",
+  "doc_link": "https://docs.datadoghq.com/integrations/launchdarkly/",
```
Comment (gzussa) [inline launchdarkly/manifest.json]: `doc_link` does not exist anymore. Please check the doc again

**Pull request merged**